althea_kernel_interface/src/exit_server_tunnel.rs

```diff
@@ -33,12 +34,19 @@ impl dyn KernelInterface {
         let mut client_pubkeys = HashSet::new();
 
         for c in clients.iter() {
+            let i_ipv6 = &c.internet_ipv6_list;
+            let mut allowed_ips = c.internal_ip.to_string().to_owned();
```

Comment:
a short comment about how this includes the internal ipv4 and then builds the ipv6 list would be good. I really like the implementation here, it's clean.

Reply:
fixed